README.md

```diff
@@ -1,13 +1,78 @@
-# Java IOs - Practical - Audio files management
-The goal of this project is to manipulate audio file through a CLI 
+# Java Audio file utility
+This is a small, cross-platform command line utility to modify and play audio files. The supported file formats are : `AIFC`, `AIFF`, `AU`, `SND`, `WAVE`. 
 
-The main features are 
-- more or less sound volume
-- speedup or slowdown track
+The features include: 
+- Changing volume
+- Changing speed
+- Play audio
 
-over the whole track
+## How to use
+This is the help message displayed when either you enter a wrong command or enter the `-h` or  `--help` command. 
+```bash
+Usage: java-ios-1.0-SNAPSHOT.jar [-hV] <filename> [COMMAND]
+A small CLI to modify audio files.
+      <filename>   The name of the file.
+  -h, --help       Show this help message and exit.
+  -V, --version    Print version information and exit.
+Commands:
+  play    Play audio file.
+  volume  Change volume on audio file.
+  speed   Change speed on audio file.
+```
+For the volume and speed commands, you will need to add a `-i=` which corresponds to the intensity. 
 
-Here are some idea if more time
-- apply the features over only a part of the track
+### Example commands
+**Play**: 
+```bash
+java -jar your_executable_path.jar your_audio_file.wav play
+```
+This will Play your audio file from your terminal. 
 
+**Speed**: 
+```bash
+java -jar your_executable_path.jar your_audio_file.wav speed -i=1.5
+```
+Intensity takes positive floating point values. In the example here, it will speed up the audio file by 1.5 times. This will output a file in your current directory which will be called `original_file_name_x1.5.wav` for this example. 
 
+**Volume**: 
+```bash
+java -jar your_executable_path.jar your_audio_file.wav volume -i=2.7
+```
+This will make the audio louder by 2.7 times. Intensity can take any floating point value. 
+
```

Comment:
Add :  This will output a file in your current direcrory which will be called 'original_file_name_at_1.5_volume.wav' for this example. 
and also add : 
Negative values will make a the audio softer.